dbus/org.sailfishos.sfmf.ufs.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Interface=org.sailfishos.sfmf.ufs
+Name=org.sailfishos.sfmf.ufs
```

Comment:
sfmf is the project name ("Sailfish Manifest"). "ufs" = "upgrade factory snapshot", i thought org.sailfishos.sfmf.upgradefactorysnapshot" might be a bit too long of a name. Any suggestions? Maybe `org.sailfishos.slipstream.upgrade` and `org.sailfishos.slipstream.unpack`? Does "slipstream" say more about what it does than "sfmf"?